modules/releasing/pages/tenant-release-pipelines.adoc

```diff
@@ -123,7 +123,7 @@ roleRef:
  name: publish-role <.>
 subjects:
 - kind: ServiceAccount
-  name: appstudio-pipeline <.>
+  name: build-pipeline-$COMPONENT_NAME <.>
```

Comment:
This is for tenant pipelines which run in the tenant workspace. But in this workspace the `build-pipeline-$COMPONENT_NAME` will already exist, so you won't be able to create it, no? So maybe for this example, you can use release-service-account (applies to the other change in this file as well)?

Reply:
Why would `release-service-account` exist in a tenant namespace?

Reply:
changing to : release-service-account

Reply:
I think `build-pipeline-$COMPONENT_NAME` like you had it made more sense here. `release-service-account` won't be in a tenant namespace, and the build one will be, so it is a lot less complex for users to use that. I guess we can wait for Martin to respond why he wanted this change

Reply:
yeah please @mmalina @johnbieren agree on something ;)

Reply:
> Why would release-service-account exist in a tenant namespace?

Sorry, I misread it. I thought there was a snippet to create a service account, but it's just a subject for the rolebinding. So yeah, the build one makes more sense.

Reply:
ok changed it back in this doc to 
build-pipeline-$COMPONENT_NAME